controllers/githubHooks.js

```diff
@@ -28,7 +28,7 @@ var HooksController = {
       }
 
       // Begin the webhook decoding
-      var body = JSON.parse(req.body.payload);
```

Comment:
I'm not really sure, I was going to try and investigate, but I'm pretty unfamiliar with this.